doc/project/release_process.rst

```diff
@@ -91,29 +94,58 @@ At that point, the whole process starts over again.
 
 .. _release_quality_criteria:
 
-Release Quality Criteria
-************************
-
-The current backlog of prioritized bugs shall also be used as a quality metric
-to gate the final release. The following counts shall be used:
-
-.. csv-table:: Bug Count Release Thresholds
-   :header: "High", "Medium", "Low"
-   :widths: auto
+Release Criteria
+****************
+
+The main motivation is to clearly have the criteria in place that must be met
+for a release. This will help define when a release is "done" in terms that most
+people can understand and in ways that help new people to understand the process
+and participate in creating successful releases:
+
+- The release criteria documents all the requirements of our target audience for
+  each Zephyr release
+- The target audiences for each release can be different, and may overlap
+- The criteria at any given time are not set in stone: there may be requirements
+  that have been overlooked, or that are new, and in these cases, the criteria
+  should be expanded to ensure all needs are covered.
+
+Below is the high level criteria to be met for each release:
+
+- No blocker bugs / blocking issues
+- All relevant tests shall pass on ``Tier 0`` platforms
+- All relevant tests shall pass on Tier 0 and 1 platforms (at least 1 per
```

Comment:
not sure about the use of ` ``Tier 0 `` ` formatting is appropriate, but in any case it should be consistent across the document